ui/src/app/utils/index.tsx

```diff
@@ -190,7 +190,8 @@ export function reactStyles<T extends {[key: string]: React.CSSProperties }>(t:
 export class ReactWrapperBase implements OnChanges, OnInit, OnDestroy {
   @ViewChild('root') rootElement: ElementRef;
 
-  constructor(private WrappedComponent: React.ComponentType, private propNames: string[]) {}
+  constructor(private WrappedComponent: React.ComponentType | React.FunctionComponent,
```

Comment:
Hm.. `ComponentType` is already a union of `ComponentClass<P> | FunctionComponent<P>`. Did this not compile?

Reply:
Must have been a leftover from an experiment